doc/developers-guide/accumulators.md

```diff
@@ -140,9 +146,10 @@ The stanza can be sent or dropped, or it may error out - whatever it does this f
 Under the hood the routing modules work by passing an original accumulator and a stanza which is to be sent, which may be the original
 xml element, same element with modifications, or a broadcast.
 
-Eventually there are three things that may happen: either the stanza is sent to another c2s process - in this case we produce
-another fresh accumulator out of the data to be sent, stripping it of all caches and the likes but preserving original ref and
-timestamp (`mongoose_acc:strip/2`), or stanza is stored as offline, or is sent out to another XMPP server via `ejabberd_s2s`.
+Eventually there are three things that may happen:
+* the accumulator is sent to another c2s process, pubsub node or muc room - in this case we produce another fresh accumulator out of the data to be sent, stripping it of all caches and the likes but preserving original ref and timestamp and those attrs that should be carried on (`mongoose_acc:strip/2`)
```

Comment:
the accumulator is sent to another c2s process, pubsub node or muc room - in this case we produce another fresh accumulator out of the data to be sent, stripping it of all caches and the likes but preserving the original ref and timestamp and those attrs that should be carried on (`mongoose_acc:strip/2`)